pkg/infra/kubernetes/exec_unit.go

```diff
@@ -3,11 +3,12 @@ package kubernetes
 import (
 	"errors"
 	"fmt"
+	autoscaling "k8s.io/api/autoscaling/v2beta2"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
nit: would it be possible to standardize to `gopls` import ordering and grouping? I believe `goimports` is the equivalent that intellij can use